sympy/algebras/quaternion.py

```diff
@@ -454,6 +470,17 @@ def _ln(self):
     def pow_cos_sin(self, p):
         """Computes the pth power in the cos-sin form.
 
+        Parameters
+        ==========
+
+        p: int
```

Comment:
I think it should be `p : int`, as it is not rendered in correct format

<img src="https://user-images.githubusercontent.com/34944973/52392094-cdc9ad80-2ae3-11e9-803b-d12cb676c45c.png" width="300">